assets/js/util.js

```diff
@@ -162,6 +164,11 @@ $(document).ready(function () {
         return false;
     });
 
+    $('#originalText').mouseup(function () {
```

Comment:
> Is there not a resize event?

This [stackoverflow link](https://stackoverflow.com/questions/5570390/resize-event-for-textarea) discusses about a `resize` event that we will need to code up to detect a change in `textarea` dimensions. This too, uses `mouseup` event only! Thus, I chose to go with the `mouseup` approach!